packages/experimental/Button/src/Button.types.ts

```diff
@@ -47,6 +47,11 @@ export interface ButtonCoreTokens extends LayoutTokens, FontTokens, IBorderToken
    */
   spacingIconContent?: number;
 
+  /**
+   * The amount of spacing between "after" icon and the content
+   */
+  afterIconSpacing?: number;
```

Comment:
Do we need to have a separate token for this? Could we reuse spacingIconContent?

Reply:
> > > No full review yet, but adding the `<Text>` tags around the text (which is what would be expected in RN) would probably make implementing this behavior harder. Would probably be something like "check type of this React node. If Text, grab its children and wrap it in the Slot to stylize it"
> > 
> > 
> > Do we even need to make it a slot? If we pass in, say, FURN Text, then it should just get all the right styling/theme info by default?
> 
> Hmm I don't think so. We don't have default settings for the standalone Text component, and the Text color you would want would differ based on the appearance of the button. Ex. Text color for a primary button would be different from text color for the default button.
> 
> Without moving away from compose, slots are the only way to pass theme token info from theme to the inner render.

Currently it's implemented using slot. I didn't change the styles, I only added check if children[idx] is string and wrapped it in the existing Text slot.

Reply:
> > > > No full review yet, but adding the `<Text>` tags around the text (which is what would be expected in RN) would probably make implementing this behavior harder. Would probably be something like "check type of this React node. If Text, grab its children and wrap it in the Slot to stylize it"
> > > 
> > > 
> > > Do we even need to make it a slot? If we pass in, say, FURN Text, then it should just get all the right styling/theme info by default?
> > 
> > 
> > Hmm I don't think so. We don't have default settings for the standalone Text component, and the Text color you would want would differ based on the appearance of the button. Ex. Text color for a primary button would be different from text color for the default button.
> > Without moving away from compose, slots are the only way to pass theme token info from theme to the inner render.
> 
> If FURN Text had default settings, would we then be able to pass it in as a child? Note I don't quite like this solution, it requires the client to _know_ to use FURN Text over the standard RN Text when using FURN Button, which would be really easy to do wrong.

With current solution by default we can write <Button>Text</Button> and put <Text> inside if we want. We can put any children components